cellom2tif/filetypes.py

```diff
@@ -1,3 +1,32 @@
+import os
+
+
+def fn_has_ext(fn, ext, case_sensitive=False):
```

Comment:
That looks good to me.

I'm also reconsidering the need for the ``case_sensitive`` parameter, I can't think of a use case where someone would want ``DIB`` files and not ``dib`` files. Any objections to removing it?